Cleanup docker tutorial (#551)
foundriesio/docs: source/tutorials/getting-started-with-docker/gs-docker-compose.rst

Docker Compose YAML
^^^^^^^^^^^^^^^^^^^

> The YAML file ``docker-compose.yml`` defines services, networks, and volumes for multi-container Docker applications.

Reviewer: isn't this like too long?

Author: Yes and no. There are a couple rules/suggestions that influence line length. Semantic newlines with sentences less than 25 words > column 100. So if semantic newlines are not used, then column 100 is the cutoff.

Author: But I do see a way to shorten the sentence, which I will do.

> The YAML file ``docker-compose.yml`` defines services, networks, and volumes for multi-container Docker applications.

Author: ```suggestion
``docker-compose.yml`` defines the services, networks, and volumes for multi-container Docker apps.
```

           MSG: "${MSG-Hello world}"

Most of the parameters were already used in the previous commands. 
The only thing to change is the image parameter.

> In the next tutorial, you will build and deploy the image with FoundriesFactory®, where the image with ``hub.foundries.io`` will be necessary.

Author: ```suggestion
In the next tutorial, you will build and deploy with FoundriesFactory®, where ``hub.foundries.io`` will be necessary.
```

> As you are still developing locally, you need to edit the image parameter to use the image and tag built during the previous steps, ``shellhttpd:1.0``:

Author: ```suggestion
As you are still developing locally, edit the image parameter to use the image and tag from the previous steps, ``shellhttpd:1.0``:
```
